src/twisted/protocols/amp.py

```diff
@@ -608,7 +609,7 @@ class IncompatibleVersions(AmpError):
 PROTOCOL_ERRORS = {UNHANDLED_ERROR_CODE: UnhandledCommand}
 
 
-class AmpBox(dict):
+class AmpBox(Dict[bytes, bytes]):
```

Comment:
We really should have landed type annotations as a separate, prior change. They're not related to "AMPv2".

Reply:
I know, sorry for the noise, I wanted to follow trunk closely with tests passing.